pom.xml

```diff
@@ -28,11 +28,14 @@
 
   <groupId>com.huaweicloud</groupId>
   <artifactId>spring-cloud-huawei</artifactId>
-  <version>1.3.2</version>
+  <version>1.3.3-Hoxton-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>Spring Cloud Huawei</name>
 
   <properties>
+    <java.version>1.8</java.version>
+    <spring-cloud.version>Greenwich.SR5</spring-cloud.version>
+    <spring-boot.version>2.1.6.RELEASE</spring-boot.version>
   </properties>
```

Comment:
need change to:
`<spring-cloud.version>Hoxton.SR3</spring-cloud.version>`
`<spring-boot.version>2.2.6.RELEASE</spring-boot.version>`

Reply:
will fix in next PR . This PR only change version.